app/components/InstanceManagerModal/Settings/Settings.js

```diff
@@ -14,145 +14,162 @@ import { history } from '../../../store/configureStore';
 import ForgeManager from './ForgeManager';
 
 const FormItem = Form.Item;
-type Props = {};
 
 let watcher = null;
 
-class Instances extends Component<Props> {
-  props: Props;
+function Instances(props) {
+  const [instanceConfig, setInstanceConfig] = useState(null);
+  const [checkingForge, setCheckingForge] = useState(true);
+  const [unMounting, setUnMounting] = useState(false);
+  const [globalArg, setglobalArg] = useState();
+  const [OverArgs, setOverArgs] = useState(true);
```

Comment:
Variables names should be camel-cased

Reply:
only the one on the 24 and 35 lines are mine